templates/application-cert-manager.yaml

```diff
@@ -24,9 +24,9 @@ spec:
         maxDuration: 3m0s
       limit: 5
   source:
-    repoURL: 'https://helm.gpkg.io/platform'
-    chart: glueops-cert-manager
-    targetRevision: 0.9.1
+    repoURL: 'https://github.com/GlueOps/platform-helm-chart-cert-manager.git'
```

Comment:
Never thought about doing this. @fernandoataoldotcom this is easier than doing all the tags/build we do.